tests/common/dummy_data.rs

```diff
@@ -75,26 +86,34 @@ pub async fn add_dummy_data(test_env: &TestEnvironment) -> DummyData {
 }
 
 pub async fn add_project_alpha(test_env: &TestEnvironment) -> (Project, Version) {
+    add_public_dummy_project("alpha", DummyJarFile::DummyProjectAlpha, test_env).await
+}
+
+pub async fn add_project_beta(test_env: &TestEnvironment) -> (Project, Version) {
     // Adds dummy data to the database with sqlx (projects, versions, threads)
     // Generate test project data.
     let json_data = json!(
         {
-            "title": "Test Project Alpha",
-            "slug": "alpha",
+            "title": "Test Project Beta",
+            "slug": "beta",
             "description": "A dummy project for testing with.",
-            "body": "This project is approved, and versions are listed.",
+            "body": "This project is not-yet-approved, and versions are draft.",
             "client_side": "required",
             "server_side": "optional",
             "initial_versions": [{
-                "file_parts": ["dummy-project-alpha.jar"],
+                "file_parts": ["dummy-project-beta.jar"],
```

Comment:
These may as well use the new DummyJarBeta constants as well

Reply:
Good idea, will do

Reply:
Didn't go as far as making the beta create project use the function yet because of the differences between them, but we should try and get that at some point. It uses the DummyJar enum now, though.

Reply:
Sounds good, I can have a look at this in my PR as I'm updating the dummy data code anyway